<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtomicWidget</class>
 <widget class="QWidget" name="AtomicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableView" name="offerBookTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>BTC change address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="change_address">
         <property name="text">
          <string>bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>XMR receive address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xmr_address">
         <property name="text">
          <string>888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnjyPbb3iQ1YBRk1UXcdRsiKc9dhwMVgN5S9cQUiyoogDavup3H</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_swap">
       <property name="text">
        <string>Swap</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_refreshOffer">
       <property name="text">
        <string>Refresh Offer Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_addRendezvous">
       <property name="text">
        <string>Add Rendezvous Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="meta_label">
       <property name="text">
        <string>No swap tool configured, click configure to set up</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_configure">
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>fromChanged(QString)</slot>
  <slot>toChanged(QString)</slot>
  <slot>toComboChanged(QString)</slot>
 </slots>
</ui>
